src/views/forms/AccountManagementForms/WithdrawForm.tsx

```diff
@@ -329,6 +334,12 @@ export const WithdrawForm = () => {
           },
         });
       }
+      if (
+        !debouncedAmountBN.isZero() &&
+        MustBigNumber(debouncedAmountBN).lte(MIN_CCTP_TRANSFER_AMOUNT)
+      ) {
+        return 'Amount must be greater than 10 USDC';
```

Comment:
nit would be localization and using MIN_CCTP_TRANSFER_AMOUNT! but all good as followup